presentation/build.gradle

```diff
@@ -37,6 +38,7 @@ dependencies {
     compile project(path: ':domain')
     compile "com.jakewharton.rxrelay2:rxrelay:2.0.0"
     testCompile "com.nhaarman:mockito-kotlin:$mockito_version"
+    kapt "com.android.databinding:compiler:$android_gradle_version"
```

Comment:
was this required here? There shouldn't be any BindingAdapters in the presentation layer, only fields.  There isn't anything that should have to be generated.